Containerfile

```diff
@@ -10,6 +10,8 @@ ENV GO_COMPLIANCE_DEBUG=0
 # downloading the latest version
 ENV GOTOOLCHAIN=local
 
+ENV GOEXPERIMENT=strictfipsruntime
```

Comment:
@mansikulkarni96  Isn't the  `Containerfile`  used only by konflux? if so, consider overriding the e2e test jobs.

Reply:
yes, the konflux test isn't passing. Looking into it.